dev-lang/mujs/mujs-0_p20161031.ebuild

```diff
@@ -0,0 +1,32 @@
+# Copyright 1999-2016 Gentoo Foundation
+# Distributed under the terms of the GNU General Public License v2
+# $Id$
+
+EAPI=6
+
+inherit eutils flag-o-matic toolchain-funcs vcs-snapshot
+
+DESCRIPTION="lightweight Javascript interpreter"
+HOMEPAGE="http://mujs.com/"
+SRC_URI="http://git.ghostscript.com/?p=mujs.git;a=snapshot;h=a0ceaf5050faf419401fe1b83acfa950ec8a8a89;sf=tgz -> ${P}.tar.gz"
+
+LICENSE="AGPL-3"
+SLOT="0"
+KEYWORDS="~alpha amd64 ~arm ~hppa ~ia64 ~ppc ~ppc64 ~sparc ~x86 ~amd64-linux ~ppc-macos ~x64-macos ~x86-macos"
+IUSE=""
+
+DEPEND=""
+RDEPEND="${DEPEND}"
+
+src_prepare() {
+	default
+	eapply "${FILESDIR}"/${PN}-0_p20150202-Makefile.patch
+	# workaround for linkage of app-text/mupdf-1.7a
+	# TODO: generate a shared library and IUSE=static-libs
+	append-cflags -fPIC
+	tc-export CC
+}
+
+src_install() {
+	emake DESTDIR="${ED}" install
```

Comment:
Actually, try removing `src_install` completely. It seems that the default should work fine.